server/services/interventionsService.test.ts

```diff
@@ -0,0 +1,50 @@
+import { pactWith } from 'jest-pact'
+import InterventionsService from './interventionsService'
+import config from '../config'
+
+pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, provider => {
+  let interventionsService: InterventionsService
+
+  beforeEach(() => {
+    const testConfig = { ...config.apis.interventionsService, url: provider.mockService.baseUrl }
```

Comment:
do we need the extra url param here. the url is inside the config structure - can we just change it there?

Reply:
sorry - i read this wrong, looks good as is.